src/main/java/com/unbappd/enterprise/service/ReviewServiceStub.java

```diff
@@ -0,0 +1,32 @@
+package com.unbappd.enterprise.service;
+
+import com.unbappd.enterprise.dto.Review;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class ReviewServiceStub implements IReviewService {
+
+
+    public List<Review> reviewsDAO = new ArrayList<>();
+
+    public ReviewServiceStub() {
+
+    }
+
+    @Override
+    public void save(Review review) {
+        reviewsDAO.add(review);
+    }
+
+    @Override
+    public Review fetchById(int placeID) {
+        Review review = new Review();
+        review.setReviewText("A fabulous beer");
+        review.setPlacesId(9);
+        return review;
```

Comment:
Here I am adding code to the fetchById so you can use this service in your unit tests. This is a service stub so the information is hardcoded. It is not a list, but it is a start for getting unit tests to work and using unit tests to guide you through making your app.